packages/devtools/src/commands/acceptAlert.js

```diff
@@ -0,0 +1,8 @@
+export default async function acceptAlert () {
+    if (!this.activeDialog) {
+        throw new Error('no such alert')
+    }
+
+    await this.activeDialog.accept()
+    return null
```

Comment:
Why return `null`everywhere, can't we return undefined like it does by default if you do not return anything?

Reply:
Please see the [protocol](https://w3c.github.io/webdriver/), it all of these cases it says:

> Return success with data null.